### PR TITLE
hw 01 Chloe Hans-Barrientos

### PR DESCRIPTION
**Pull request by chloehansb**
I first forked the assignment from our course GitHub into my GitHub and then I cloned it from my GitHub to my laptop to allow me to work on it. I then worked on it in RStudio using the following Markdown grammar: bullet points, headers, bold, and embedded an image. Once I finished creating my biography, I committed the README.md file and pulled and pushed the file to my GitHub and then I pulled the request on GitHub to send the file to our class GitHub and then I edited the title and provided a description of my process. For some reason when I pulled the request, my homework assignment did not immediately show up on our class GitHub so I got a bit frustrated. I kept refreshing the class GitHub and trying to pull the request and couldn't figure out what had gone wrong. It then all of a sudden showed up on the class GitHub but seems like it had a delayed reaction.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,20 @@
 # hw01
 Repository for homework 1
+
+#**About Me**
```
Comment (csava1025) [inline README.md:4]: I think here you needed to put a space between the # and the ** in order to create the header. I had the same issue with mine initially! The same should apply for the other headers that you have!

**Review by csava1025 — COMMENTED** [review]
Hi! I don't think I accepted the request to join Github in time, so I'm just picking random ones. I thought yours looked great! It was creative, you incorporated images and lists successfully and tried out all of the different features of markdown. The only suggestion I had was to include a space between the pound sign and the text for the header next time so that it prints!

**Review by hibyeun — COMMENTED** [review]
You did a great job with inserting photo and making bullet points but as mentioned in the other reviewer's comments, a space is needed to make a proper header.

**Comment by cernhofer** [comment]
Evaluation HW-01 for Chloe Hans-Barrientos
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |      x     |              |            |
| **Coding strategy**         |           |        x    |            |
| **Presentation**            |           |       x      |            |
| **Achievement, creativity** |           |    x         |            |
| **Ease of access**          |     x      |              |            |

* Good work! For the most part, your readme was organized and clear and you've used a number of Markdown elements.
* Make sure to preview your file on github before turning it in- it looks like you forgot to add spaces in your headers after the pound sign (#)- this prevented it from compiling properly.

**Comment by ayseguluysal** [comment]
Evaluation HW-1 for Chloe Hans Barrientos
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |   X        |              |            |
| **Coding strategy**         |           |      X        |            |
| **Presentation**            |      X     |              |            |
| **Achievement, creativity** |    X       |              |            |
| **Ease of access**          |      x     |              |            |

I really enjoyed reading your biography, and really good job with the picture. Only comment that I would make is don't forget to add a space between # sign and your text next time.

**Review by ayseguluysal — COMMENTED** [review]
Your READ.me was easy to follow, and you incorporated a lot of Markdown elements! Great job!

**Pull request closed without merging**